pkgs/tools/filesystems/mergerfs/default.nix

```diff
@@ -1,18 +1,19 @@
-{ stdenv, fetchgit, fuse, pkgconfig, which, attr, pandoc, git }:
+{ stdenv, fetchgit, fuse, pkgconfig, which, attr, pandoc, git, autoconf, automake, libiconv, libtool, gettext }:
 
 stdenv.mkDerivation rec {
   name = "mergerfs-${version}";
-  version = "2.16.1";
+  version = "2.22.1";
 
   # not using fetchFromGitHub because of changelog being built with git log
   src = fetchgit {
     url = "https://github.com/trapexit/mergerfs";
     rev = "refs/tags/${version}";
-    sha256 = "12fqgk54fnnibqiq82p4g2k6qnw3iy6dd64csmlf73yi67za5iwf";
+    sha256 = "1rw9jxzryn687ivfis44aaflqhk77arm71q8hzlrs8lnjk83fn20";
     deepClone = true;
+    leaveDotGit = true;
```

Comment:
I got a different sha256 and probably this option is the fault. Is it really required for the build?